tests/clover.xml

```diff
@@ -0,0 +1,1461 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is this file pushed on purpose? Looks like a coverage report that is probably subject to change with each code change, no?